sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Azure.Iot.TimeSeriesInsights.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
+    <AssemblyTitle>Azure Time Series Insights SDK</AssemblyTitle>
+    <Version>1.0.3-beta.19</Version>
```

Comment:
Why this version?

Reply:
I believe this is what I copied from the template. I will update to 1.0.0-beta-1